src/containers/Sidebar/Sidebar.test.jsx

```diff
@@ -421,6 +429,35 @@ describe('<Sidebar />', () => {
     }
   });
 
+  it('hides highlights when we have groups with a subset of all learners', async () => {
+    getConfig.mockReturnValue({ FEATURE_CONTENT_HIGHLIGHTS: true });
+    const store = mockStore({
+      ...initialState,
+      portalConfiguration: {
+        enterpriseFeatures: {
+          enterpriseGroupsV1: true,
+        },
+      },
+    });
+
+    LmsApiService.fetchEnterpriseGroup.mockImplementation(() => Promise.resolve({
+      data: { results: { applies_to_all_contexts: false } },
```

Comment:
From [how you're using the results of fetchEnterpriseGroup](https://github.com/openedx/frontend-app-admin-portal/pull/1191/files#diff-f6bf4f6edd2dac680925dc1421e37fa202dad8b65415291bded6349d4c679ec5R67), should be returning a group array here.
```suggestion
      data: { results: [{ applies_to_all_contexts: false }] },
```